packages/react/src/Pagination/model.tsx

```diff
@@ -148,13 +148,14 @@ export function buildComponentData(
       key = 'page-prev'
       content = 'Previous'
       if (page.disabled) {
-        Object.assign(props, {as: 'button', 'aria-disabled': 'true'})
+        Object.assign(props, {'aria-disabled': 'true'})
```

Comment:
We can add `role="link"` here, since I believe we'd only need it when the link is disabled. This should allow us to remove it below in the else condition